src/generators/operationGenerator.ts

```diff
@@ -708,10 +704,11 @@ function writeSendOperationRequest(
   operationMethod: MethodDeclaration,
   operationSpecName: string,
   clientDetails: ClientDetails,
-  isInline = false
+  isInline = false,
+  sendParams: string
```

Comment:
Where are we passing this? Also, did this give you a compile error? I thought optional parameters can't be followed by required ones

Reply:
I just saw where you are passing this, I see you are passing `""` is that expected or WIP?